electrum/plugin.py

```diff
@@ -232,6 +232,8 @@ def run_hook(name, *args):
                 results.append(r)
 
     if results:
+        if name == "wallet_info_buttons":
```

Comment:
not sure If I understand this constraint - why can only one result from hook be accepted? Here I expect more than one hook to provide data